devnet/README.md

```diff
@@ -52,26 +52,25 @@ There are four Docker files used in the devnet.
 
 ### cmd/update-dd-agent-confd
 
-A command line app to create a `confd.yml` file used by the Datadog agent when piping metrics to Datadog.  It's used in the both `alice.Dockerfile` and `bob.Dockerfile` to create specific `confd.yml` files.
+A command line app to create a `confd.yml` file used by the Datadog agent when piping metrics to Datadog. It's used in the both `alice.Dockerfile` and `bob.Dockerfile` to create specific `confd.yml` files.
 
 ### alice.node.key
 
-This key is injected in `alice.Dockerfile` so it uses the same public key for the `bootnodes` param in `bob.Dockerfile`. 
+This key is injected in `alice.Dockerfile` so it uses the same public key for the `bootnodes` param in `bob.Dockerfile`.
 
 ### docker-compose.yml
 
-The Docker Compose file.  Specifies the IP addresses of all the nodes. 
-
+The Docker Compose file. Specifies the IP addresses of all the nodes.
 
 ## ECS Fargate Deployment
 
-The [`docker-compose.yml`](gssmr-ecs/docker-compose.yml) file within `devnet/gssmr-ecs` folder uses Docker Compose ECS plugin to deploy and update an existing AWS ECS Cluster using the Fargate launch type running a Gossamer devnet with 3 services corresponding to the 3 keys used `alice`, `bob`, and `charlie`.  
+The [`docker-compose.yml`](gssmr-ecs/docker-compose.yml) file within `devnet/gssmr-ecs` folder uses Docker Compose ECS plugin to deploy and update an existing AWS ECS Cluster using the Fargate launch type running a Gossamer devnet with 3 services corresponding to the 3 keys used `alice`, `bob`, and `charlie`.
```

Comment:
nit a lot of auto-formatting fixes, would it be possible to save without formatting and open another PR fixing all markdown documents? Otherwise it'll eventually be strange in the git history/git blame why those lines got modified here.

Reply:
updated